README.md

```diff
@@ -46,37 +50,21 @@ In order to run against any cluster, use the following:
 ```sh
 $ mvn clean integration-test  \
     -Dspark-distro-tgz=spark/spark-2.3.0-SNAPSHOT-bin.tgz  \
-    -DextraScalaTestArgs="-Dspark.kubernetes.test.master=k8s://https://<master> -Dspark.docker.test.driverImage=<driver-image> -Dspark.docker.test.executorImage=<executor-image>"
-```
-
-# Preserve the Minikube VM
-
-The integration tests make use of
-[Minikube](https://github.com/kubernetes/minikube), which fires up a virtual
-machine and setup a single-node kubernetes cluster within it. By default the vm
-is destroyed after the tests are finished.  If you want to preserve the vm, e.g.
-to reduce the running time of tests during development, you can pass the
-property `spark.docker.test.persistMinikube` to the test process:
-
-```
-$ mvn clean integration-test  \
-    -Dspark-distro-tgz=spark/spark-2.3.0-SNAPSHOT-bin.tgz  \
-    -DextraScalaTestArgs=-Dspark.docker.test.persistMinikube=true
+    -DextraScalaTestArgs="-Dspark.kubernetes.test.master=k8s://https://<master>
```

Comment:
Is the master parameter now required? If so, explain how people can get it from a minikube?

Reply:
`kubectl cluster-info` is already discussed above.

Reply:
Master parameter shouldn't be required.

Reply:
...But this section is under the "running against an arbitrary cluster" section, so in this context the parameter would be required.